src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -24,6 +25,17 @@
 {
     protected $container;
 
+    private $currentDir;
```

Comment:
can be made protected, and setCurrentDir be removed IMO (also required in my other PR)